### ISSUE TITLE
Implement top right 3 dots menu in profiles

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw-hf#32]
The top right 3 dots menu will be dependent on the type of profile the client has:

All clients, where {wra}=0 and {age}>5 years, is not in a register:
Registration info
Malaria diagnosis

All clients, where {wra} =1, is not in a register:
Registration info
Pregnancy confirmation
FP initiation
Malaria diagnosis

Client in Malaria register:
Registration info
Malaria follow-up

Client in Child register:
Registration info
Sick child follow-up
Malaria diagnosis

Client in Child and Malaria register:
Registration info
Sick child follow-up
Malaria follow-up

Client in ANC register:
Registration info
ANC danger signs outcome
Pregnancy outcome

Client in ANC and Malaria register:
Registration info
ANC danger signs outcome
Pregnancy outcome
Malaria diagnosis

Client in PNC register:
Registration info
PNC danger signs outcome

Client in PNC and Malaria register:
Registration info
PNC danger signs outcome
Malaria diagnosis

Client in FP register:
Registration info
FP method change/stop

Client in FP and Malaria register:
Registration info
FP method change/stop
Malaria diagnosis

**Comment by rowo** [comment]
@msschroeder Yes agreed, editing should be allowed if you can add. Can you think of any cons to allowing this?

**Comment by cafootitt** [comment]
Is the registration form the exact same form as in the CHW app?
